From d4021487539b0f7758ec644f2e0d83df95c66bba Mon Sep 17 00:00:00 2001
From: Michael Allman <michael@videoamp.com>
Date: Sat, 16 Apr 2016 17:23:59 -0700
Subject: [PATCH] PARQUET-581: Fix two instances of the conflation of the min
 and max row

count for page size check in ParquetOutputFormat.java

Author: Michael Allman <michael@videoamp.com>

Closes #340 from mallman/fix_minmax_conflation and squashes the following commits:

79331a5 [Michael Allman] PARQUET-581: Fix two instances of the conflation of the min and max row count for page size check in ParquetOutputFormat.java
---
 .../java/org/apache/parquet/hadoop/ParquetOutputFormat.java   | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java b/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java
index 6accce17c5..6cfa8e93c3 100644
--- a/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java
+++ b/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java
@@ -252,7 +252,7 @@ public static int getMinRowCountForPageSizeCheck(Configuration configuration) {
 
   public static int getMaxRowCountForPageSizeCheck(Configuration configuration) {
     return configuration.getInt(MAX_ROW_COUNT_FOR_PAGE_SIZE_CHECK,
-        ParquetProperties.DEFAULT_MINIMUM_RECORD_COUNT_FOR_CHECK);
+        ParquetProperties.DEFAULT_MAXIMUM_RECORD_COUNT_FOR_CHECK);
   }
 
   public static boolean getEstimatePageSizeCheck(Configuration configuration) {
@@ -383,7 +383,7 @@ public RecordWriter<Void, T> getRecordWriter(Configuration conf, Path file, Comp
     if (INFO) LOG.info("Maximum row group padding size is " + maxPaddingSize + " bytes");
     if (INFO) LOG.info("Page size checking is: " + (props.estimateNextSizeCheck() ? "estimated" : "constant"));
     if (INFO) LOG.info("Min row count for page size check is: " + props.getMinRowCountForPageSizeCheck());
-    if (INFO) LOG.info("Min row count for page size check is: " + props.getMaxRowCountForPageSizeCheck());
+    if (INFO) LOG.info("Max row count for page size check is: " + props.getMaxRowCountForPageSizeCheck());
 
     WriteContext init = writeSupport.init(conf);
     ParquetFileWriter w = new ParquetFileWriter(
